src/main/fc/fc_init.c

```diff
@@ -232,7 +233,7 @@ void spiPreInit(void)
     spiPreInitCs(IO_TAG(L3GD20_CS_PIN));
 #endif
 #ifdef MAX7456_SPI_CS_PIN
-    spiPreInitCsOutPU(IO_TAG(MAX7456_SPI_CS_PIN)); // XXX 3.2 workaround for Kakute F4. See comment for spiPreInitCSOutPU.
+    spiPreInitCsOutPU(max7456Config()->csTag); // XXX 3.2 workaround for Kakute F4. See comment for spiPreInitCSOutPU.
```

Comment:
The comment should probably go for this line.

Reply:
This is a rogue occurrence of target name outside of target directory :-D

Reply:
@mikeller The workaround is still there. I'll just remove the "3.2".

@DieHertz lol

Reply:
Argh... You are right. This is for kakuta that uses level shifter to interface max chip, which can only be pulled up by explicit PP output.

Reply:
Ah yes, I remember. Anything we can / have to do about this for universal target support?

I see the potential for damage if a Max7456 CS is accidentally assigned to a pin that is an input. But since the universal targets don't have pin assignments, this would have to be done on purpose, which pretty much falls into the same category as flashing an incorrect target - all bets are off in that case.

We _could_ introduce a modifier for CS pins to allow the config to specify if a logical 1 out is required. This would make it possible to activate this only for the board(s) that need it, and potentially do it for other peripherals if they use level shifters. But it would introduce quite a bit of additional complexity.